tests/import_test.go

```diff
@@ -0,0 +1,35 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	testSuiteName           = "Importer Test Suite"
+	namespacePrefix         = "importer"
+	waitForCDIImportTimeout = 30 * time.Second
+)
+
+var _ = Describe(testSuiteName, func() {
+	f, err := framework.NewFramework(namespacePrefix)
+	if err != nil {
+		Fail("Unable to create framework struct")
+	}
+
+	Describe("Creating standard PVC should not trigger importer", func() {
+		It("PVC should remain empty after being created", func() {
+			pvc, err := f.CreatePVCFromDefinition(utils.NewPVCDefinition("no-import", "1G", nil, nil))
+			Expect(err).ToNot(HaveOccurred())
+			// Wait a while to see if CDI puts anything in the PVC.
+			time.Sleep(waitForCDIImportTimeout)
+			Expect(framework.VerifyPVCIsEmpty(f, pvc)).To(BeTrue())
+			// Not deleting PVC as it will be removed with the NS removal.
```

Comment:
I also know you prefer me to delete this PVC explicitly as well. I explained that I prefer to let the NS cascade the removal because its fewer requests to the API server, and the internal delete of the NS is faster than doing everything explicitly.

Reply:
Right. See my comment above and then decide how you want to do it.

Reply:
Right its about run time vs ease of debugging... I am going to go with using the cascade from the NS delete for now. I might change my mind in the future.